### PR TITLE
add databricks case study

### PR DESCRIPTION
**Pull request by anotherrachel**
Signed-off-by: Yajing Wang <yajing.wang@pingcap.com>

### CODE REVIEW

File: how-databricks-tackles-the-scalability-limit-with-a-mysql-alternative.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+logo: /images/blog/customers/databricks-logo.png
+image: /images/blog/how-databricks-tackles-the-scalability-limit-with-a-mysql-alternative.png
+notShowOnLogoWall: true
+---
+
+**Industry:** Information Technology
+
+**Author:** Min Zhou (Staff engineer at Databricks)
+
+**Editor:** [Yajing Wang](https://github.com/anotherrachel), Tom Dewan
```
Comment (ran-huang) [inline how-databricks-tackles-the-scalability-limit-with-a-mysql-alternative.md:18]:
```suggestion
**Editors:** [Yajing Wang](https://github.com/anotherrachel), Tom Dewan
```

File: how-databricks-tackles-the-scalability-limit-with-a-mysql-alternative.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+### Storage
+
+Cloud providers usually offer both local storage physically attached to the machine as well as remote disks. According to our benchmarks, the local disk and the remote disk have the same read performance. Although local disks offer the best write latency, they add significant overhead to operations. For example, If we replace a host, we must migrate all the data on it. In some extreme scenarios, if we update all the machines in the cluster, we must copy all the data on that cluster to another fleet. Furthermore, because local disks are ephemeral, a local disk configuration cannot tolerate concurrent maintenance of machines.
+
+To reduce the overhead of remote disks, we propose a cost-effective solution. This also improves the throughput of the remote disk and lowers the latency to an acceptable level.
+
+### Service Isolation
```
Comment (ran-huang) [inline how-databricks-tackles-the-scalability-limit-with-a-mysql-alternative.md:118]: Sentence case？

**Review by ran-huang — APPROVED** [review]

File: how-databricks-tackles-the-scalability-limit-with-a-mysql-alternative.md (hunk 2 above)
Comment (anotherrachel) [reply how-databricks-tackles-the-scalability-limit-with-a-mysql-alternative.md:118]:
```suggestion
### Service isolation
```

File: how-databricks-tackles-the-scalability-limit-with-a-mysql-alternative.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
+---
+title: How Databricks Tackles the Scalability Limit with a MySQL Alternative
+author: ['Min Zhou']
+date: 2021-10-07
```
Comment (anotherrachel) [inline how-databricks-tackles-the-scalability-limit-with-a-mysql-alternative.md:4]:
```suggestion
date: 2021-10-04
```

**Pull request merged**